xrmocap/model/registrant/smplify.py

```diff
@@ -637,6 +705,28 @@ def __compute_loss__(self,
 
         return losses
 
+    def __post_process_loss__(self, losses: dict, **kwargs) -> dict:
```

Comment:
According to [PEP8](https://peps.python.org/pep-0008/#descriptive-naming-styles), starting with a single under line is better. The style I used before is unprofessional, and we can correct it from now on.
```suggestion
    def _post_process_loss(self, losses: dict, **kwargs) -> dict:
```